### PR TITLE
Add custom keystore and refactor HypixelApi.java

### PR DESCRIPTION
**Pull request by romangraef**

### CODE REVIEW

**Review by Ascynx — APPROVED** [review]
~~lessgo~~, should probably close connection after it's not used anymore (see infer check).

**Review by Lulonaut — APPROVED** [review]

**Comment by Mqisty** [comment]
looks god to me ! :fire: :+1: :100:

**Review by Ascynx — DISMISSED** [review]
:scream: :scream: :scream: :scream:

**Review by Mqisty — DISMISSED** [review]
looks bad to me ! :fire_extinguisher: :-1: :1234:

**Review by Lulonaut — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewer.java [hunk 1]
```diff
@@ -825,6 +887,10 @@ public JsonArray getSkyblockProfiles(Runnable runnable) {
 								}
 
 								String cuteName = profile.get("cute_name").getAsString();
+								if (profile.has("selected") && profile.get("selected").getAsBoolean()) {
+									lastCuteName = cuteName;
+									break;
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewer.java:892]: Remove the break as it makes the profile not show in the profile selector

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]

**Review by hannibal002 — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**